CMakeLists.txt

```diff
@@ -0,0 +1,126 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.10 FATAL_ERROR)
+
+# Set a default build type if none was specified
+SET(default_build_type "Release")
+IF(EXISTS "${CMAKE_SOURCE_DIR}/.git")
+    SET(default_build_type "Debug")
```

Comment:
Maybe we should just default to Release always, most people will probably clone this and want Release by default

Reply:
Release is now the default always